dotnet/src/SemanticKernel/KernelBuilder.cs

```diff
@@ -22,17 +22,18 @@ namespace Microsoft.SemanticKernel;
 /// </summary>
 public sealed class KernelBuilder
 {
-    private KernelConfig _config = new();
     private Func<ISemanticTextMemory> _memoryFactory = () => NullMemory.Instance;
     private ILoggerFactory _loggerFactory = NullLoggerFactory.Instance;
     private Func<IMemoryStore>? _memoryStorageFactory = null;
-    private IDelegatingHandlerFactory? _httpHandlerFactory = null;
+    private IDelegatingHandlerFactory _httpHandlerFactory = NullHttpHandlerFactory.Instance;
     private IPromptTemplateEngine? _promptTemplateEngine;
     private readonly AIServiceCollection _aiServices = new();
 
     private static bool _promptTemplateEngineInitialized = false;
     private static Type? _promptTemplateEngineType = null;
 
+    public IDelegatingHandlerFactory HttpHandlerFactory => this._httpHandlerFactory;
```

Comment:
As mentioned above, I'm not sure it's required. Otherwise, it will also need XML documentation.

Reply:
I expected a WithDHFactory or something on Builder such that the factory could be passed similar to config to other object initialization.

Reply:
It's not, thanks for the good sight!